src/lib/block_producer/block_producer.ml

```diff
@@ -1241,6 +1225,9 @@ let run ~context:(module Context : CONTEXT) ~vrf_evaluator ~prover ~verifier
 let run_precomputed ~context:(module Context : CONTEXT) ~verifier ~trust_system
     ~time_controller ~frontier_reader ~transition_writer ~precomputed_blocks =
   let open Context in
+  let log_bootstrap_mode () =
```

Comment:
why it was re-introduced locally?

Reply:
Nice catch, it was a mistake, I removed it